meet_balls_app/templates/base.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>MeetBalls</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/static/css/styles.css" rel="stylesheet" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-md navbar bg-warning">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="/">MeetBalls</a>
+        <button
+          class="navbar-toggler"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#navbarNav"
+          aria-controls="navbarNav"
+          aria-expanded="false"
+          aria-label="Toggle navigation"
+        >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+          <ul class="navbar-nav ms-auto">
+            <li class="nav-item">
+              <a class="nav-link" href="/game-events">Games</a>
+            </li>
```

Comment:
The href will redirect the user to /game-events url which isn't defined right now.
It's on perpose so when game_events template PR will be pushed it will be defined.

if it's problematic the way it is right now lem me know, i will change it to "#" so it wont go crazy on us